### PR TITLE
Support global link definitions.

### PR DESCRIPTION
**Pull request by ypid**
Works by symlinking the global.rst which is included in this repo to all
submodules which have a `docs/includes` directory in them. This allows
to update the definition in on central place.

Wrote scripts to generate:

* Ansible modules
* DebOps roles (likes to GitHub repo)
* DebOps Contrib roles (likes to GitHub repo)
* ypid roles (likes to GitHub repo, seamless self plug. Added my private Ansible roles for reference.)

Usage for roles (or other DebOps repos):

Create `./docs/includes/all.rst`:
```rst
.. include:: includes/global.rst
```

Ensure line present in `.gitignore`:
```rst
docs/includes/global.rst
```

Optional: copy https://github.com/debops/docs/tree/master/docs/includes/global.rst
to `docs/includes/global.rst`
(Only needed when the repos should be build as standalone for testing.)

Use in docs like so (example `docs/getting-started.rst`):
```rst
Getting started
===============

.. contents::
   :local:

.. include:: includes/all.rst

DebOps_ rocks!
..    ↑ makes it a hyperlink.
```

That should be all. Enjoy :wink:

Implements: https://github.com/debops/docs/issues/155

### CODE REVIEW

File: docs/includes/40gen_debops_repos.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+.. _debops.ferm: https://github.com/debops/ansible-ferm
+.. _debops.gitlab: https://github.com/debops/ansible-gitlab
+.. _debops.gitlab_ci: https://github.com/debops/ansible-gitlab_ci
+.. _debops.gitlab_ci_runner: https://github.com/debops/ansible-gitlab_ci_runner
+.. _debops.gitusers: https://github.com/debops/ansible-gitusers
+.. _debops.ifupdown: https://github.com/debops/ansible-ifupdown
+.. _debops.java: https://github.com/debops/ansible-java
+.. _debops.kvm: https://github.com/debops/ansible-kvm
+.. _debops.lxc: https://github.com/debops/ansible-lxc
+.. _debops.mailman: https://github.com/debops/ansible-mailman
```
Comment (drybjed) [inline docs/includes/40gen_debops_repos.rst]: The list of repositories is incompliete, what's going on here? Issues with GitHub API?

**Comment by drybjed** [comment]
This looks very impressive. :-) Since role tests include documentation testing, addition of this includes will require support for this in the `assert_docs` function in `test-suite` repository.

**Comment by ypid** [comment]
@drybjed Thanks very much :wink: I will update `assert_docs` later today.

File: docs/includes/40gen_debops_repos.rst (hunk 1 above)
Comment (ypid) [inline docs/includes/40gen_debops_repos.rst]: Spot on :+1: Fixed. I will finish this later today.

**Pull request merged**